src/widgets/Tabs/PatientEdit.js

```diff
@@ -76,13 +74,11 @@ const PatientEditComponent = ({
             inputConfig={getFormInputConfig('patient', currentPatient)}
           />
         </View>
-        <View style={localStyles.verticalSeparator} />
-        <View style={localStyles.formContainer}>
-          {surveySchema && (
-            <JSONForm ref={formRef} onSubmit={onSubmitSurvey} surveySchema={surveySchema}>
-              <View />
-            </JSONForm>
-          )}
+        <View style={surveySchema ? localStyles.verticalSeparator : localStyles.hidden} />
```

Comment:
yeah, I don't like it either.
not showing the JSONForm throws a forwardRef warning though, and this fixes it 🤷‍♂️